### ISSUE TITLE
Add REST API

### ISSUE THREAD

**Issue by jn8** [randomcolour/randomcolourr#23]

**Comment by mattmalin** [comment]
Nice - I suggest moving the API out to separate file in R directory as plumber.R though :)

**Comment by jn8** [comment]
Yes I was thinking about that and did that as well but not committed. I
wanted to test if plumber would break the tests.

Automated testing is such a revelation!

Jonathan

Best Selling Instructor - Data Analytics in Excel & R

Check out a free sample course at www.DataStrategyWithJonathan.com
